lib/arke/command.rb

```diff
@@ -12,10 +12,27 @@ def run!
     module_function :run!
 
     def load_configuration
-      configs = YAML.load(File.open('config/variables.yaml'))
+      strategy = YAML.load_file('config/strategy.yaml')['strategy']
+
+      drivers = {
+        'bitfinex' => Arke::Exchange::Bitfinex,
+        'rubykube' => Rubykube::MarketApi,
```

Comment:
Todo, rubykube should also be an exchange
Arke::Exchange::Rubykube < BaseExchange